lib/routes.js

```diff
@@ -0,0 +1,5 @@
+FlowRouter.route('/',{
+  action: function(){
+    FlowLayout.render('frontlayout',{main:'frontbody'});
```

Comment:
@JuliaKoe you are passing frontbody as a parameter to your "main". this is good, but on the layout, you are not "accepting" this "main"!